config/reek.yml

```diff
@@ -65,6 +65,7 @@ TooManyInstanceVariables:
   exclude:
   - Mutant::Mutator # 4 vars
   - Mutant::Parallel::Master # 4 vars
+  - Mutant::Integration::Rspec # 4 vars
```

Comment:
Yeah, ideally reek (just like rubocop and others) would not only allow me to add a whitelist, but allow me to specify the whitelisted value.

Right now those exceptions could grow to 1k of ivars without being constrainted.